packages/excel/src/row.ts

```diff
@@ -89,9 +93,44 @@ export abstract class Row {
 		return subrowOffset + SUBROW_HEADER_SIZE + baseOffset
 	}
 
-	protected unknown(opts?: ColumnSeekOptions) {
-		// NOOP - retrieve the column definition to advance to the next column position
-		this.getColumnDefinition(opts)
+	protected unknown(
+		opts?: ColumnSeekOptions,
+	): {type: ColumnDataType; value: string | number | bigint | boolean | null} {
+		const definition = this.getColumnDefinition(opts)
+		let value: string | number | bigint | boolean | null
+		switch (definition.dataType) {
+			case ColumnDataType.BOOLEAN:
+			case ColumnDataType.PACKED_BOOL_0:
+			case ColumnDataType.PACKED_BOOL_1:
+			case ColumnDataType.PACKED_BOOL_2:
+			case ColumnDataType.PACKED_BOOL_3:
+			case ColumnDataType.PACKED_BOOL_4:
+			case ColumnDataType.PACKED_BOOL_5:
+			case ColumnDataType.PACKED_BOOL_6:
+			case ColumnDataType.PACKED_BOOL_7:
+				value = this.boolean(opts)
+				break
+			case ColumnDataType.STRING:
+				value = this.string(opts)
+				break
+			case ColumnDataType.INT_8:
+			case ColumnDataType.UINT_8:
+			case ColumnDataType.INT_16:
+			case ColumnDataType.UINT_16:
+			case ColumnDataType.INT_32:
+			case ColumnDataType.UINT_32:
+			case ColumnDataType.FLOAT_32:
+				value = this.number(opts)
+				break
+			case ColumnDataType.INT_64:
+			case ColumnDataType.UINT_64:
+				value = this.bigint(opts)
+				break
+			default:
+				value = null
```

Comment:
the cases above should ideally be comprehensive, if we're missing a column data type that's a pretty catastrophic issue that'll crop up in >1 place.

if you add the ol' unreachable check to the utilities file:

```ts
class UnreachableError extends Error {
	constructor(value: never) {
		super(`Unreachable value received: ${value}`)
	}
}
```

then you can `throw new UnreachableError(value)` here and ts'll ensure the switch is kept up to date

plus you'll be able to remove the `null` from the return type which is comfy